src/app/explore/explore.component.ts

```diff
@@ -124,6 +130,16 @@ export class ExploreComponent implements OnDestroy, OnInit {
   }
 
   onScroll() {
+    if (this.scroller.getScrollPosition()[1] < this.smallestSignificantScrollPx) {
```

Comment:
I had to look up what [1] means, so:
```suggestion
    const scrollPositionY = this.scroller.getScrollPosition()[1];
    if (scrollPositionY < this.smallestSignificantScrollPx) {
```